src/components/02-organisms/ButtonAppBar/index.js

```diff
@@ -132,7 +120,10 @@ const ButtonAppBar = ({
                 backgroundColor: "transparent"
               }}
             >
-              {formatMessage(authMessages.logOut)} ({user.username})
+              {intl.formatMessage(authMessages.logOut)} ({user.username}) <br />{" "}
+              <span className="text-gray-600">
+                {R.path(["nimi", "fi"], organisaatio)}
```

Comment:
Organization name based on language selection?

Reply:
Will be fixed by replacing "fi" with intl.locale.